plugins/dynamix/include/LocalMaster.php

```diff
@@ -0,0 +1,21 @@
+<?PHP
+/* Copyright 2015, Bergware International.
+ * Copyright 2015, Lime Technology
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ */
+?>
+<?
+$lmb = exec("nmblookup -M -- -|grep -Pom1 '^\S+'");
+$tag = exec("nmblookup -A $lmb|grep -Pom1 '^\s+\K\S+'");
+if (exec("ifconfig|grep -PA1 '^(bond|eth|br)\d+:'|grep -om1 'inet $lmb'")) {
```

Comment:
Ronald, the `grep -om1 'inet $lmb'` in this line will find a matching IP from the local machine that *starts with* the Master Browser IP address (e.g. $lmb = 10.0.0.1  and  eth0 = 10.0.0.10).  To fix, just add a space after $lmb:
`grep -om1 'inet $lmb '`